.pre-commit-config-slow.yaml

```diff
@@ -0,0 +1,9 @@
+# Slow pre-commit checks we don't want to run locally with each commit.
+
+exclude: 'src/pip/_vendor/'
```

Comment:
I don't think we need to exclude for this one.

Reply:
Indeed. Removed. Thanks for the review.